.travis.yml

```diff
@@ -1,13 +1,10 @@
 sudo: true
 language: node_js
-before_install:
-  - if [[ "$TRAVIS_OS_NAME" == "linux" ]]; then sudo apt-get update; fi
-  - if [[ "$TRAVIS_OS_NAME" == "linux" ]]; then sudo apt-get install chromium-browser; fi
-  - if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then brew update; fi
-  - if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then brew cask install chromium; fi
```

Comment:
Should this configuration be left in so when we add macOS support again, it’s easier?